### DISCUSSION TITLE
Love how you display the endpoint to test right there

### DISCUSSION THREAD

**Discussion by monica-m-ps** [badges/shields#9541] [General] [answered]
<img width="410" alt="image" src="https://github.com/badges/shields/assets/94380874/15b74043-3caa-4c8a-968a-152d6f2468a9">

I love this layout. Can you please share how did you do it? Are you using a plugin? I see the content for this page is in js and not in markdown. I'm a technical writer and would love to know how you implemented this. Thanks

**Comment by jNullj** [comment, marked as answer]
I think this is made using [Docusaurus OpenAPI](https://github.com/cloud-annotations/docusaurus-openapi#readme) plugin.

**Reply by chris48s** [reply]
Yep. We generate an OpenAPI spec from the shields URL schema and then use docusaurus-openapi (with a couple of customisations) to render it.
